### PR TITLE
KER-325 : Change LOG level on InvalidationExoCache

### PR DESCRIPTION
**Pull request by aboughzela**

### CODE REVIEW

File: exo.kernel.component.cache/src/main/java/org/exoplatform/services/cache/invalidation/InvalidationExoCache.java [hunk 1]
```diff
@@ -168,15 +168,15 @@ public void removeLocal(Serializable key) throws NullPointerException
    public void put(K key, V value) throws NullPointerException
    {
       HashCode hashCode = new HashCode<V>(value);
-      LOG.info(getName() + " PUT KEY " + key+ " hash "+ hashCode.hashCode());
+      LOG.debug(getName() + " PUT KEY " + key+ " hash "+ hashCode.hashCode());
```
Comment (vsellier) [inline exo.kernel.component.cache/src/main/java/org/exoplatform/services/cache/invalidation/InvalidationExoCache.java]: Please use string substitution "{}.."

File: exo.kernel.component.cache/src/main/java/org/exoplatform/services/cache/invalidation/InvalidationExoCache.java [hunk 2]
```diff
@@ -413,14 +413,14 @@ public void onPut(CacheListenerContext context, K key, HashCode<V> obj) throws E
...
-            LOG.info(getName() + " ON PUT equal KEY " + key+ " hash "+ obj.hashCode());
+            LOG.debug(getName() + " ON PUT equal KEY " + key+ " hash "+ obj.hashCode());
 
             // We assume that it is the same value so we don't change the value in the cache
             value = currentValue;
          }
          else
          {
-            LOG.info(getName() + " ON PUT not equal KEY " + key+ " hash "+ obj.hashCode());
+            LOG.debug(getName() + " ON PUT not equal KEY " + key+ " hash "+ obj.hashCode());
```
Comment (vsellier) [inline exo.kernel.component.cache/src/main/java/org/exoplatform/services/cache/invalidation/InvalidationExoCache.java]: Please use string substitution "{}.."

File: exo.kernel.component.cache/src/main/java/org/exoplatform/services/cache/invalidation/InvalidationExoCache.java [hunk 3]
```diff
@@ -413,14 +413,14 @@ public void onPut(CacheListenerContext context, K key, HashCode<V> obj) throws E
          V currentValue = localCache.get(key);
          if (currentValue != null && obj != null && currentValue.hashCode() == obj.hashCode())
          {
-            LOG.info(getName() + " ON PUT equal KEY " + key+ " hash "+ obj.hashCode());
+            LOG.debug(getName() + " ON PUT equal KEY " + key+ " hash "+ obj.hashCode());
```
Comment (vsellier) [inline exo.kernel.component.cache/src/main/java/org/exoplatform/services/cache/invalidation/InvalidationExoCache.java]: Please use string substitution "{}.."

File: exo.kernel.component.cache/src/main/java/org/exoplatform/services/cache/invalidation/InvalidationExoCache.java [hunk 4]
```diff
@@ -168,15 +168,15 @@ public void removeLocal(Serializable key) throws NullPointerException
...
+      LOG.debug(getName() + " PUT KEY " + key+ " hash "+ hashCode.hashCode());
       delegate.put(key, hashCode);
    }
 
    @Override
    public void putLocal(K key, V value) throws NullPointerException
    {
       HashCode hashCode = new HashCode<V>(value);
-      LOG.info(getName() + " PUT LOCAL KEY " + key+ " hash "+ hashCode.hashCode());
+      LOG.debug(getName() + " PUT LOCAL KEY " + key+ " hash "+ hashCode.hashCode());
```
Comment (vsellier) [inline exo.kernel.component.cache/src/main/java/org/exoplatform/services/cache/invalidation/InvalidationExoCache.java]: Please use string substitution "{}.."

File: exo.kernel.component.cache/src/main/java/org/exoplatform/services/cache/invalidation/InvalidationExoCache.java (hunk 1 above)
Comment (aboughzela) [reply exo.kernel.component.cache/src/main/java/org/exoplatform/services/cache/invalidation/InvalidationExoCache.java]: ok

File: exo.kernel.component.cache/src/main/java/org/exoplatform/services/cache/invalidation/InvalidationExoCache.java (hunk 4 above)
Comment (aboughzela) [reply exo.kernel.component.cache/src/main/java/org/exoplatform/services/cache/invalidation/InvalidationExoCache.java]: ok

**Pull request merged**